handbook/people.md

```diff
@@ -267,6 +267,13 @@ At the end of the quarter, and at key points throughout the quarter (every 3 wee
 - OKRs they are finalized 1 week from when topdown OKRs were initially reviewed.
 - Finalized OKRs are shared company-wide and at "all hands" and, at least to some degree, become public.
 
+## Workversaries
+
+We're happy you've ventured a trip around the sun with Fleet. Let's celebrate!
+
+- Every week, People Operations reviews Fleeties start dates.
```

Comment:
I've been lower-casing "fleeties" elsewhere, so let's keep it the same for consistency.

cc @Desmi-Dizney and @mike-j-thomas

Reply:
We need a possessive here as well, so `fleeties'`

Reply:
I added `fleeties` and `workiversary` to our style guide. Although I would prefer to consistently treat a `Fleetie` celebrating their workiversary as a proper noun, the same as I would an `Austinite` or `Londoner` drinking boba or tea, respectively, and fix up any past instances as we come across them. 
<img width="1175" alt="image" src="https://user-images.githubusercontent.com/78363703/162125307-dfc2c944-c061-4123-a207-017458391412.png">

Reply:
@mike-j-thomas Up to you, Mike